website_and_docs/content/documentation/webdriver/interactions/print_page.en.md

```diff
@@ -177,4 +181,41 @@ Using `getBackground()` and `setBackground()` methods, you can get/set whether t
 {{< tab header="Kotlin" >}}
 {{< badge-implementation >}}
 {{< /tab >}}
+{{< /tabpane >}}
+
+## Printing
+
+Once you've configured your PrintOptions, you're ready to print the page. To do this, 
+you can invoke the print function, which generates a PDF representation of the web page. 
+The resulting PDF can be saved to your local storage for further use or distribution. 
+Using `PrintsPage()`, the print command will return the PDF data in base64-encoded format, which can be decoded 
+and written to a file in your desired location, and using `BrowsingContext()` will return a String. `BrowsingContext()`
+is part of Selenium's BiDi implementation.
```

Comment:
may be making a note will be useful like

**Note**:  `BrowsingContext()` is part of Selenium's BiDi implementation. To enable BiDi see "Enabling BiDi" see https://www.selenium.dev/documentation/webdriver/bidi/

Any thoughts on this @shbenzer !

Reply:
I like that - I'll add it in the morning

Reply:
Added